CIS Scripts/3_Security_Remediation.sh

```diff
@@ -474,37 +472,42 @@ if [ "$Audit2_7_1" = "1" ]; then
 	echo "$(date -u)" "2.7.1 remediated" | tee -a "$logFile"
 fi
 
-# 2.8 Pair the remote control infrared receiver if enabled
+# 2.8 Disable "Wake for network access"
```

Comment:
Curiosity question. Are these rules being renamed or replaced?

Reply:
There are some renames, but most non-functional changes are just rules being moved around by CIS. You can find the different benchmarks at https://www.cisecurity.org/benchmark/apple_os/